src/wkwebkit.cs

```diff
@@ -1025,9 +1030,29 @@ interface WKWebView
 		[NullAllowed, Export ("underPageBackgroundColor", ArgumentSemantic.Copy)]
 		UIColor UnderPageBackgroundColor { get; set; }
 
-		[iOS (15,4), MacCatalyst (15,4), Mac (12,3), NoTV]
+		[iOS (16,0), MacCatalyst (16,0), Mac (13,0), NoTV]
 		[Export ("fullscreenState")]
 		WKFullscreenState FullscreenState { get; }
+
+		[Mac (13,0), iOS (15,5), MacCatalyst(15,5), NoWatch, NoTV]
+		[Export ("minimumViewportInset")]
+		UIEdgeInsets MinimumViewportInset { get; }
+
+		[Mac (13,0), iOS (15,5), MacCatalyst(15,5), NoWatch, NoTV]
+		[Export ("maximumViewportInset")]
+		UIEdgeInsets MaximumViewportInset { get; }
+
+		[Mac (13,0), iOS (15,5), MacCatalyst(15,5), NoWatch, NoTV]
+		[Export ("setMinimumViewportInset:maximumViewportInset:")]
+		void SetViewportInsets (UIEdgeInsets minimumViewportInset, UIEdgeInsets maximumViewportInset);
+
+		// [iOS (16,0), MacCatalyst (16,0), NoMac, NoWatch, NoTV]
+		// [Export ("findInteractionEnabled")]
+		// bool FindInteractionEnabled { [Bind ("isFindInteractionEnabled")] get; set; }
+
+		// [iOS (16,0), MacCatalyst (16,0), NoMac, NoWatch, NoTV]
```

Comment:
Please make a separated issue about this if you haven't already and link it [here](https://github.com/xamarin/xamarin-macios/issues/15232) so we do not forget